content/shared/v3-line-protocol.md

```diff
@@ -284,13 +284,197 @@ but with a different field set, the field set becomes the union of the old
+
+#### Append-only with unique timestamps (recommended)
+
+Write each change as a new point with a unique timestamp using the actual event time.
+Query for the most recent point to get the current value.
+
+**Line protocol example**:
+
+```text
+device_status,device_id=sensor01 status="active",temperature=72.5 1700000000000000000
+device_status,device_id=sensor01 status="active",temperature=73.1 1700000300000000000
+device_status,device_id=sensor01 status="inactive",temperature=73.1 1700000600000000000
+```
+
+**SQL query to get latest state**:
+
+```sql
+SELECT
+  device_id,
+  status,
+  temperature,
+  time
+FROM device_status
+WHERE time >= now() - INTERVAL '7 days'
+  AND device_id = 'sensor01'
+ORDER BY time DESC
+LIMIT 1
+```
+
+**InfluxQL query to get latest state**:
+
+```influxql
+SELECT LAST(status), LAST(temperature)
+FROM device_status
+WHERE device_id = 'sensor01'
+  AND time >= now() - 7d
+GROUP BY device_id
+```
+
+#### Append-only with change tracking field
+
+If you need to filter by "changes since a specific time," add a dedicated `last_change_timestamp` field.
+
+**Line protocol example**:
+
+```text
+device_status,device_id=sensor01 status="active",temperature=72.5,last_change_timestamp=1700000000000000000i 1700000000000000000
+device_status,device_id=sensor01 status="active",temperature=73.1,last_change_timestamp=1700000300000000000i 1700000300000000000
+device_status,device_id=sensor01 status="inactive",temperature=73.1,last_change_timestamp=1700000600000000000i 1700000600000000000
+```
+
+**SQL query to get changes since a specific time**:
+
+```sql
+SELECT
+  device_id,
+  status,
+  temperature,
+  time
+FROM device_status
+WHERE last_change_timestamp >= 1700000000000000000
+ORDER BY time DESC
+```
+
+### Anti-patterns to avoid
+
+The following patterns will produce non-deterministic results when duplicate points are flushed together:
+
+#### ❌ Overwriting the same (time, tags) point
+
+**Don't do this**:
+
+```text
+-- All writes use the same timestamp
+device_status,device_id=sensor01 status="active",temperature=72.5 1700000000000000000
+device_status,device_id=sensor01 status="active",temperature=73.1 1700000000000000000
+device_status,device_id=sensor01 status="inactive",temperature=73.1 1700000000000000000
+```
+
+**Problem**: When these writes are flushed together, any of the three values might be retained.
+
+#### ❌ Adding a field while overwriting the same (time, tags)
+
+**Don't do this**:
+
+```text
+-- All writes use the same timestamp, but add a version field
+device_status,device_id=sensor01 status="active",temperature=72.5,version=1i 1700000000000000000
+device_status,device_id=sensor01 status="active",temperature=73.1,version=2i 1700000000000000000
+device_status,device_id=sensor01 status="inactive",temperature=73.1,version=3i 1700000000000000000
+```
+
+**Problem**: The points are still duplicates (same time and tags).
+Adding a field doesn't make them unique points.
```

Comment:
````suggestion
Adding a field doesn't make a points unique.
Points with the same time and tag set are still considered duplicates--for example,
**don't do this**:

```text
-- All writes use the same timestamp, but add a version field
device_status,device_id=sensor01 status="active",temperature=72.5,version=1i 1700000000000000000
device_status,device_id=sensor01 status="active",temperature=73.1,version=2i 1700000000000000000
device_status,device_id=sensor01 status="inactive",temperature=73.1,version=3i 1700000000000000000
```
````